aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -161,6 +161,70 @@ module AptosFramework::Stake {
         missed_votes: vector<u64>,
     }
 
+    /// The events emitted for the entire StakePool's lifecycle.
+    struct StakePoolEvents has key {
+        register_validator_candidate_events: EventHandle<RegisterValidatorCandidateEvent>,
+        set_operator_events: EventHandle<SetOperatorEvent>,
+        add_stake_events: EventHandle<AddStakeEvent>,
+        rotate_consensus_key_events: EventHandle<RotateConsensusKeyEvent>,
+        increase_lockup_events: EventHandle<IncreaseLockupEvent>,
+        join_validator_set_events: EventHandle<JoinValidatorSetEvent>,
+        distribute_rewards_events: EventHandle<DistributeRewardsEvent>,
+        unlock_stake_events: EventHandle<UnlockStakeEvent>,
+        withdraw_stake_events: EventHandle<WithdrawStakeEvent>,
+        leave_validator_set_events: EventHandle<LeaveValidatorSetEvent>,
+    }
+
+    struct RegisterValidatorCandidateEvent has drop, store {
+        pool_address: address,
+    }
+
+    struct SetOperatorEvent has drop, store {
+        pool_address: address,
+        old_operator: address,
+        new_operator: address,
+    }
+
+    struct AddStakeEvent has drop, store {
+        pool_address: address,
+        amount: u64,
```

Comment:
the name should be clearer about whether it's the total amount or the added amount

Reply:
Updated